CoreMapView/src/au/gov/asd/tac/constellation/views/mapview2/utilities/Vec3.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2010-2022 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.mapview2.utilities;
+
+/**
+ * Utility class to hold 3 variables and do vector maths on them
+ *
+ * @author altair1673
+ */
+public class Vec3 {
+    private double x = 0;
+    private double y = 0;
+    private double z = 0;
```

Comment:
similar case as previous, add these default instantiations to the no parameter constructor